src/tests/home/HowSection.spec.js

```diff
@@ -0,0 +1,20 @@
+// import expect from 'expect';
+// import React from 'react';
+// import { mount, shallow } from 'enzyme';
+// import HowSection from '../../js/components/home/HowSection';
+
+// const HowSectionFn = () => {
+//   return shallow(<HowSection />);
+// }
+
+// describe('HowSection', () => {
+//   const HowSectionRender = HowSectionFn();
+
+//   it('loads markup', () => {
+//     expect(HowSectionRender.find('how').length).toEqual(1);
+//   });
+
+//   it('displays accurate heading', () => {
+//     expect(HowSectionRender.find('section-heading').text()).toEqual('Partner with us to help students succeed in a future of technology');
+//   });
+// });
```

Comment:
Yeah you are right.... I could not run the tests for these two components because these two components depend on `svg` and `jpeg` files respectively and compiling these components for testing kept throwing errors